internal/pkg/daemon/bpfrecorder/bpfrecorder.go

```diff
@@ -657,26 +655,13 @@ func (b *BpfRecorder) findSystemMountNamespace() (uint64, error) {
 	return uint64(ns), nil
 }
 
-func (b *BpfRecorder) trackProfileForPid(pid uint32, mntns uint64, profile interface{}) {
-	comm := b.commForPid(pid)
```

Comment:
This method is now unused and can be removed.

Reply:
ok deleted